gym/wrappers/common/frame_skipping.py

```diff
@@ -0,0 +1,79 @@
+import gym
+
+__all__ = ['Skip', 'Skip1', 'Skip2', 'Skip3', 'Skip4', 'Skip5', 'Skip6', 'Skip7', 'Skip8', 'Skip9', 'Skip10']
+
+class Skip(gym.Wrapper):
+    """
+        Generic common frame skipping wrapper
+        Will perform action for `x` additional steps
+    """
+    def __init__(self, env, repeat_for_x_steps):
+        super(Skip, self).__init__(env)
+        self.repeat_count = repeat_for_x_steps
+        self.record_stepcount = not hasattr(env.unwrapped, 'stepcount')
+        if self.record_stepcount:
+            env.unwrapped.stepcount = 0
+
+    def _step(self, action):
+        done = False
+        total_reward = 0
+        current_step = 0
+        while current_step < (self.repeat_count + 1) and not done:
+            if self.record_stepcount:
+                self.env.unwrapped.stepcount += 1
+            obs, reward, done, info = self.env.step(action)
+            total_reward += reward
+            current_step += 1
+        return obs, total_reward, done, info
+
+    def _reset(self):
+        if self.record_stepcount:
+            self.env.unwrapped.stepcount = 0
+        return self.env.reset()
+
+    @property
+    def stepcount(self):
```

Comment:
I think better to expose this as a key on `info`. I would probably do it as `skip.stepcount`.